ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/Buffer.cpp

```diff
@@ -20,48 +20,53 @@
 
 #include "Buffer.h"
 
-#include "Map.h"
-#include "MapQuickView.h"
 #include "GeometryEngine.h"
-#include "GraphicsOverlay.h"
 #include "Graphic.h"
+#include "GraphicListModel.h"
+#include "GraphicsOverlay.h"
+#include "GraphicsOverlayListModel.h"
 #include "LinearUnit.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "MapTypes.h"
+#include "Point.h"
+#include "Polygon.h"
 #include "SimpleFillSymbol.h"
 #include "SimpleLineSymbol.h"
 #include "SimpleMarkerSymbol.h"
 #include "SimpleRenderer.h"
-#include "MapTypes.h"
-#include "GraphicsOverlayListModel.h"
 #include "SymbolTypes.h"
-#include "GraphicListModel.h"
-#include "Polygon.h"
-#include "Point.h"
-
-#include <cmath>
```

Comment:
This is likely needed still. We've seen build issues on Linux when we use things like NAN but don't include this